README.md

```diff
@@ -27,25 +27,28 @@ alias guarddog='docker run --rm ghcr.io/datadog/guarddog'
 
 ```sh
 # Scan the most recent version of the 'requests' package
-guarddog scan requests
+guarddog pypi scan requests
 
 # Scan a specific version of the 'requests' package
-guarddog scan requests --version 2.28.1
+guarddog pypi scan requests --version 2.28.1
 
 # Scan the 'request' package using 2 specific heuristics
-guarddog scan requests --rules exec-base64 --rules code-execution
+guarddog pypi scan requests --rules exec-base64 --rules code-execution
 
 # Scan the 'requests' package using all rules but one
-guarddog scan requests --exclude-rules exec-base64
+guarddog pypi scan requests --exclude-rules exec-base64
 
 # Scan a local package
-guarddog scan /tmp/triage.tar.gz
+guarddog pypi scan /tmp/triage.tar.gz
```

Comment:
This bothers me from a semantics standpoint - we're scanning a local package but using `guarddog pypi`. How about different scanning backends for local packages? e.g. `pypi-local` and `npm-local`?

Bonus: we don't have to guess anymore if the argument is a local path or an actual package name

Reply:
🤔 I am afraid it gets confusing if we have too much commands at one point but I agree with the complexity reduction. Need a couple brain cycles.

Reply:
agreed on using a flag today

Reply:
@christophetd let's put it in another PR, I was lookign at the flag for `verify` and this is not O(1) ^^